components/brave_welcome_ui/components/select-profile/index.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import * as S from './style'
+import Button from '$web-components/button'
+import LeftArrowSVG from '../svg/left-arrow'
+import AvatarIconSVG from '../svg/avatar-icon'
+import DataContext from '../../state/context'
+import { ViewType } from '../../state/component_types'
+import { ImportDataBrowserProxyImpl } from '../../api/import_data_browser'
+import { BravePrivacyBrowserProxyImpl } from '../../api/privacy_data_browser'
+import { getLocale } from '$web-common/locale'
+
+interface ProfileItemProps {
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void
+  profileName: string
+  id: number
+  isChecked: boolean
+}
+
+function ProfileItem (props: ProfileItemProps) {
+  return (
+    <div className="item-box">
+      <div className="item-grid">
+        <label className="item-info" htmlFor={`profile-${props.id}`}>
```

Comment:
nit: You might not neccessarily need to use `htmlFor`, and `id` attributes if you have the `label` wrap the `input`

Reply:
done